### DISCUSSION TITLE
Compatible with your obsidian-enhanced-annotations plugin

### DISCUSSION THREAD

**Discussion by michaellw** [ycnmhd/obsidian-lineage#27] [Ideas]
I tried to add a comment after the section number, but this breaks the structure of the view of the lineage. Hopefully the two will be compatible.

![image](https://github.com/ycnmhd/obsidian-lineage/assets/276357/646966cc-4f4c-4cdf-b952-b1e53435b822)
![image](https://github.com/ycnmhd/obsidian-lineage/assets/276357/a066c2a8-3d71-4ce3-b2e0-6cfdb001264d)

**Comment by ycnmhd** [comment]
Good suggestion.

Note: for future requests with clear requirements, please use `Issues` instead of `Discussions `🙏

**Reply by michaellw** [reply]
Got it👌
